views/package.json

```diff
@@ -9,6 +9,6 @@
     "url": "https://github.com/oat-sa/extension-tao-test.git"
   },
   "dependencies": {
-    "@oat-sa/tao-test-runner": "^0.2.1"
+    "@oat-sa/tao-test-runner": "^0.4.0"
```

Comment:
Let's remove the carrets and tilds for now

Reply:
fixed